py/examples/tour.py

```diff
@@ -220,35 +224,57 @@ def get_wave_completions(line, character, file_content):
+                                ui.zone('code', size=f'calc(50vh - {(header_height + mobile_blurb_height) / 2}px)'),
+                                ui.zone('preview', size=f'calc(50vh - {(header_height + mobile_blurb_height) / 2}px)'),
+                            ]),
+                    ui.zone('mobile_blurb')
+                ],
+            ),
+            ui.layout(breakpoint='m', zones=[
                 ui.zone('header'),
                 ui.zone('blurb'),
-                ui.zone('main', size='calc(100vh - 140px)', direction=ui.ZoneDirection.ROW, zones=[
-                    ui.zone('code'),
-                    ui.zone('preview')
-                ])
-            ])
+                ui.zone('main', size=f'calc(100vh - {header_height + blurb_height}px)', direction=ui.ZoneDirection.ROW,
+                        zones=[
+                            ui.zone('code'),
+                            ui.zone('preview')
+                        ])
+            ]),
         ])
+    nav_links = [
+        ('docs', 'Wave docs', 'https://wave.h2o.ai/docs/getting-started'),
+        ('discussions', 'Discussions', 'https://github.com/h2oai/wave/discussions'),
+        ('blog', 'Blog', 'https://wave.h2o.ai/blog'),
+        ('cloud', 'H2O AI Cloud', 'https://h2o.ai/platform/ai-cloud/'),
+        ('h2o', 'H2O', 'https://www.h2o.ai/'),
+    ]
     q.page['header'] = ui.header_card(
         box='header',
         title=app_title,
         subtitle=f'{len(catalog)} Interactive Examples',
         image=f'{q.app.tour_assets}/h2o-logo.svg',
         items=[
-            ui.links(inline=True, items=[
-                ui.link(label='Wave docs', path='https://wave.h2o.ai/docs/getting-started', target='_blank'),
-                ui.link(label='Discussions', path='https://github.com/h2oai/wave/discussions', target='_blank'),
-                ui.link(label='Blog', path='https://wave.h2o.ai/blog', target='_blank'),
-                ui.link(label='H2O AI Cloud', path='https://h2o.ai/platform/ai-cloud/', target='_blank'),
-                ui.link(label='H2O', path='https://www.h2o.ai/', target='_blank'),
-            ])
-        ]
-    )
+            ui.links(inline=True, items=[ui.link(label=item[1], path=item[2], target='_blank') for item in nav_links])
+        ])
+    q.page['mobile_header'] = ui.header_card(
+        box='mobile_header',
+        title=app_title,
+        subtitle=f'{len(catalog)} Interactive Examples',
+        image=f'{q.app.tour_assets}/h2o-logo.svg',
+        nav=[
+            ui.nav_group('Links', items=[ui.nav_item(name=item[0], label=item[1], path=item[2]) for item in nav_links])
+        ])
     q.page['blurb'] = ui.section_card(box='blurb', title='', subtitle='', items=[])
+    q.page['mobile_blurb'] = ui.form_card(box='mobile_blurb', items=[])
     q.page['code'] = ui.markup_card(
-        box=ui.box('code', height='calc(100vh - 140px)'),
+        box=ui.box('code', height=f'calc(100vh - {header_height + blurb_height}px)'),
```

Comment:
Is this needed if we already set the size on the layout level?
```suggestion
        box='code',
```

Reply:
In master it is like this: `box=ui.box('code', height='calc(100vh - 140px)'),`. I couldn't find a reason why. It works the same when the calculation is removed. Therefore I accept your suggestion.